### ISSUE TITLE
Java to Any translation : Array initialization

### ISSUE THREAD

**Issue by choubi90** [darrivau/GOOL#18]
``` java
int[] a={18,16,15,11,1,5,13,9};
```
is translated into: 
    ```in csharp and cpp```
``` c++
int[] a = new [];
```
``` in python```
```python
   a = (None)
```
instead of :
``` in csharp ```
``` csharp
int[] a={18,16,15,11,1,5,13,9};
```
```in cpp```
``` c++
int a[]={18,16,15,11,1,5,13,9};
```
```in python```
``` python
a =[18,16,15,11,1,5,13,9];
```

**Comment by darrivau** [comment]
java->C++ in quicksort after choubi90 modifications we have
java : int A[] = {4,1,6,8,5,2,7};
c++: int * A =  {4,1,6,8,5,2,7};
which is not a good translation. In this particular case java syntax should be kept in C++

**Comment by darrivau** [comment]
Corrected in commit 478707889
